Framework/API/test/NotebookBuilderTest.h

```diff
@@ -183,8 +181,8 @@ class NotebookBuilderTest : public CxxTest::TestSuite {
   }
 
   void test_Build_Unrolled() {
-    std::string result_markdown = R"(               "source" : "Child algorithms of TopLevelAlgorithm")";
-    std::string result_code = "               \"input\" : \"BasicAlgorithm(PropertyA='FirstOne')\",";
+    std::string result_markdown = "\t\t\t\t\t\"source\" : \"Child algorithms of TopLevelAlgorithm\"";
```

Comment:
Do we want to replace the spaces with tabs? For most cases I think we want to keep the spaces?

Reply:
This is what the jsoncpp library outputs now, in pretty much all write situations.

Reply:
Is it not configurable: https://open-source-parsers.github.io/jsoncpp-docs/doxygen/class_json_1_1_stream_writer_builder.html#a79bdf2e639a52f4e758c0b95bd1d3423 ?